Makefile

```diff
@@ -102,6 +102,19 @@ test: generate verify ## Run tests
 verify:
 	./hack/verify_boilerplate.py
 
+.PHONY: integration
+integration: generate verify ## Run integraion tests
+	bazel test --define='gotags=integration' --test_output all //test/integration/...
+
```

Comment:
```suggestion
WARNING: DO NOT SET THE `JANITOR_ENABLED` TO 1 UNLESS YOU WISH TO TEARDOWN RESOURCES IN AWS
```

Reply:
Hi @ashish-amarnath,

Are you suggesting the definition of a target named `WARNING`? If so, do you mean the text should be echoed? I am sorry, but I do not understand the proposed change.

Reply:
Sorry, I missed this comment.
The suggestion was to add a comment to caution users not to change the `JANITOR_ENABLED` flag.